Makefile

```diff
@@ -20,6 +20,12 @@ GIT_DIRTY   = $(shell test -n "`git status --porcelain`" && echo "dirty" || echo
 GO_EXE      = go
 OSNAME      = $(shell uname -o)
 ARCHNAME    = $(shell uname -m)
+GO_VERSION  = $(shell awk '/^go /{print $$2}' go.mod|head -n1)
+BUILDER_IMAGE = golang:$(GO_VERSION)-alpine
+PLATFORMS ?= linux/amd64,linux/arm64
+DOCKER_BUILDX_CMD ?= docker buildx
+IMAGE_BUILD_CMD ?= $(DOCKER_BUILDX_CMD) build
+IMAGE_GIT_TAG ?= $(shell git describe --tags --dirty --always)
```

Comment:
I have no too much history, in here https://github.com/oras-project/oras/blob/e8bca4daf111d067b4eea8fc9d89f19a6c43eb93/Makefile#L18 It is not suitable for use every time a container is built, so I make a new one